presto-hive/src/main/java/io/prestosql/plugin/hive/s3/S3SecurityMapping.java

```diff
@@ -106,6 +115,7 @@ public String toString()
                 .add("iamRole", iamRole)
                 .add("allowedIamRoles", allowedIamRoles)
                 .add("credentials", credentials)
+                .add("endpoint", endpoint)
```

Comment:
```suggestion
                .add("endpoint", endpoint.orElse(null)
```

Also we should make `toStringHelper` to skip null values. Since it also affects fields that unrelated to this commit, it  should go in separate preparatory commit.